opm/simulators/linalg/bda/opencl/openclCPR.cpp

```diff
@@ -220,7 +220,11 @@ void openclCPR<Scalar,block_size>::amg_cycle_gpu(const int level, cl::Buffer& y,
         }
 
         // solve coarsest level using umfpack
-        this->umfpack.apply(h_x.data(), h_y.data());
+        if constexpr (std::is_same_v<Scalar,float>) {
```

Comment:
Include `<type_traits>` in this file because `is_same_v` is used

Reply:
done